app/models/wpcc/legal_period_percents.rb

```diff
@@ -0,0 +1,61 @@
+module Wpcc
+  class LegalPeriodPercents
+    include ActiveModel::Model
+    attr_accessor :name,
+                  :employee_percent,
+                  :employer_percent,
+                  :tax_relief_percent
+
+    PERIODS_FILE = Wpcc::Engine.root.join(
+      'config',
+      'legal_periods_percents.yml'
+    )
+    PERIODS = HashWithIndifferentAccess.new(YAML.load_file(PERIODS_FILE))
+
+    def self.all
+      PERIODS.map do |period, percents|
+        new(
+          name: period.to_s,
+          employee_percent: percents[:employee_percent],
+          employer_percent: percents[:employer_percent],
+          tax_relief_percent: percents[:tax_relief_percent]
+        )
+      end
+    end
+
+    def self.filter(employee_percent, employer_percent)
```

Comment:
naming of `employee_percent` and `employer_percent` is confusing me a little in this file...
In this method these values are the "user input percents", right?

in `self.periods_above_user_contribution(employee_percent, employer_percent)` those same values continue with the same name. and are passed to `.below_user_contributions?`

Once we get into `def below_user_contributions?(employee_percentage, employer_percentage)` they are then named `employee_percentAGE,` at which point they are compared to `@employee_percent` which are then the values from the yaml file, i.e. the legal percents? 

Could I suggest in this method `employee_percent` and `employer_percent` be changed to something like `user_input_employee_percent` which then persists through the other methods?